hathor/transaction/resources/push_tx.py

```diff
@@ -90,6 +91,12 @@ def handle_push_tx(self, params: Dict[str, Any]) -> Dict[str, Any]:
                 'can_force': False
             }
 
+        # We are using here the method from lib because the property
+        # to identify a nft creation transaction was created on the lib
+        # to be used in the full node and tx mining service
+        tx_from_lib = lib_tx_or_block_from_bytes(tx_bytes)
```

Comment:
Shouldn't we replace the method from hathor-core with this one?

Reply:
I think replacing methods from hathor-core with the lib should be a different task. It must be done carefully because I've checked that the lib and hathor core files already have differences.

That's why I decided to use the lib method only in the new part of the code

Reply:
If they have differences, how do we know it is safe to use this new method here? I mean, applying different rules in different parts of the code might be an attack vector.

Reply:
I agree with you but we are still using the same code to validate the transactions, which is the critical part of the code. This new code here is to decide if a tx is standard or not, in order to use our public nodes APIs to push this tx.

I agree with you about the differences and we have an issue to refactor the hathor-core/hathorlib code to be only one. I can do it now if you think it's relevant but I guess it would take some days to have everything done. I think it's okay to continue with this. What do you think?

Reply:
You're right. This method is basically being used to let our public APIs check whether a tx is standard or not. I agree we can move forward. Would you open an issue in hathor-core to get this refactor done in future, please?

Reply:
Also mypy _should_ be able to complain if we're using a `hathorlib.Transaction` instead of a `hathor.transaction.Transaction` or vice-versa. But I know we can't completely rely on mypy to catch that.